### PR TITLE
Add support for MultiversX

### PR DESCRIPTION
**Pull request by joesixpack**

### CODE REVIEW

File: multiversx.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+)
+
+type ElrondResponse []struct {
+		Stake                      string  `json:"stake"`
+}
+
+func Elrond() (int, error) {
+	votingPowers := make([]big.Int, 0, 1024)
+
+        url := fmt.Sprintf("https://api.elrond.com/nodes?type=validator")
```
Comment (schimih) [inline multiversx.go:22]:
```suggestion
        url := fmt.Sprintf("https://api.multiversx.com/nodes?type=validator")
```

File: multiversx.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	"fmt"
+	"github.com/xenowits/nakamoto-coefficient-calculator/core/utils"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"net/http"
+	"sort"
+)
+
+type ElrondResponse []struct {
```
Comment (schimih) [inline multiversx.go:15]:
```suggestion
type MvxResponse []struct {
```

File: multiversx.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+		Stake                      string  `json:"stake"`
+}
+
+func Elrond() (int, error) {
```
Comment (schimih) [inline multiversx.go:19]:
```suggestion
func Multiversx() (int, error) {
```

**Comment by alfredonodo** [comment]
Hi,
any progress on this?
Thank you

**Comment by xenowits** [comment]
Closing this as https://github.com/xenowits/nakomoto-coefficient-calculator/pull/44 is merged

**Pull request closed without merging**